docs/cvm.md

```diff
@@ -1,7 +1,250 @@
+trip origin and destination purposes, land uses, and place types; and vehicle types.
+
+The geographic scope of the CVM are internal-to-internal trip movements. The market scope of the model 
+includes commercial goods movements (pickup and deliveries) as well as trips made for commercial and 
+public services. Trips made for other purposes, namely maintenance and personal, are also included in the 
+CVM if these trips are made in the context of a commercial vehicle tour pattern. The CVM explicitly 
+distinguishes between residential and non-residential customer types, and between three vehicle types—
+light, medium, and heavy—consistent with the definitions used in the Establishment Survey.
+
+The CVM does not cover the types of work-related travel that would be expected to be covered in the ABM3
+Resident model, namely workers traveling for meetings, sales calls, out-of-town travel, and similar activities.
+The CVM also does not model long-distance freight truck movements that enter and exit the region, which 
+are covered by the Heavy Truck Model (HTM).
+
+The HTM covers long-distance freight movements into and out of San Diego County. The source of the 
+demand in the HTM are commodity flows between shippers and receivers throughout North America, 
+focusing on those with either a trip end (shipper or receiver) in San Diego County or which pass through San 
+Diego County, for example, between Mexico and Los Angeles. Commodity flows are derived from the 
+Federal Highway Administration (FHWA) Freight Analysis Framework version 5 (FAF5). The model design 
+assumes that freight truck trips between establishments within San Diego County are covered by the CVM, 
+which has been designed to explicitly account for truck movements involving warehouse and distribution 
+centers and port facilities.
 
 ## Design
 
+#### Design Objectives and Overview
+The 2024 CVM design was conceived as an improvement upon the SANDAG CVM created in 2014, which 
+used a similar approach for simulating commercial vehicle tours. The new CVM design differs from the 
+previous model in several important ways. Improvement objectives of the new design include:
+
+• Better accuracy for representing total commercial vehicle demand, as represented in the new 
+surveys, particularly more accurate accounting of vehicle miles traveled (VMT) by vehicle size;
+
+• Leveraging the new survey data to capture contemporary e-commerce trends in direct deliveries 
+between warehouse fulfillment centers and consumers;
+
+• Distinguishing between residential and non-residential customer types;
+
+• Creating linkages between the CVM and the Resident model to better represent household demand 
+for food and package deliveries and for services;
+
+• Leveraging the TNC driver survey to represent this recently emerged type of on-line establishment 
+and the “gig” workers it employs.
+
+In addition, the HTM update to the most current version of FAF5 was important to representing commodity 
```

Comment:
HTM part could be removed (the last paragraph)

Reply:
Done